variables.tf

```diff
@@ -9,6 +9,21 @@ variable "enable_default_standards" {
   default     = true
 }
 
+variable "control_finding_generator" {
+  description = <<-DOC
+  Updates whether the calling account has consolidated control findings turned on.
+  If the value for this field is set to ,
+
+  SECURITY_CONTROL - Security Hub generates a single finding for a control check even when
+  the check applies to multiple enabled standards.
+
+  STANDARD_CONTROL - Security Hub generates separate findings for a control check when the
+  check applies to multiple enabled standards.
+  DOC
+  type        = string
+  default     = "SECURITY_CONTROL"
```

Comment:
It would be nice to add a validation block here to check if the values inputted are correct